R/plotCCA.R

```diff
@@ -0,0 +1,504 @@
+#' Plot RDA or CCA object
+#'
+#'
+#' @param object a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}, a
+#'   \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-constructor]{TreeSummarizedExperiment}}
+#'   or a rda/cca object. The latter is the output of \code{\link[mia:runCCA]{calculateRDA}}.
+#' 
+#' @param dimred A string or integer scalar indicating the reduced dimension to
+#'   plot. This is the output of \code{\link[mia:runCCA]{runRDA}} and resides in
+#'   \code{reducedDim(tse, dimred)}.
+#'
+#' @param colour_by String specifying the name of a column in colData to colour by
+#'   (Default: NULL).
+#' 
+#' @param color_by Alias for `colour_by`.
+#' 
+#' @param ellipse_fill if TRUE, ellipses are filled. (Default: TRUE).
```

Comment:
For consistency, check other function documents. Boolean values are described somehow like "TRUE or FALSE specifying whether to fill ellipses" or something. Try to keep these all descriptions similar